content/SCALE/SCALEUIReference/Credentials/DirectoryServices/LDAP.md

```diff
@@ -44,10 +44,10 @@ The settings on the **Basic Options** also display on the **[Advanced Options](#
 {{< truetable >}}
 | Setting | Description |  
 |---------|-------------|  
-| **Hostname** | Enter the LDAP server hostnames/IP addresses. Separate entries with <kbd>Space</kbd>. You can enter multiple hostnames/IP addresses to create an LDAP failover priority list. If a host does not respond, TrueNAS tries the next host until it establishes a connection. |
+| **Hostname** | Enter the LDAP server hostnames/IP addresses. Separate entries by pressing enter. You can enter multiple hostnames/IP addresses to create an LDAP failover priority list. If a host does not respond, TrueNAS tries the next host until it establishes a connection. |
```

Comment:
use the emphasis for keyboard by applying <kbd></kbd> tags and capitalize the key name. 
```suggestion
| **Hostname** | Enter the LDAP server hostnames/IP addresses. Separate entries by pressing <kbd>Enter</kbd>. You can enter multiple hostnames/IP addresses to create an LDAP failover priority list. If a host does not respond, TrueNAS tries the next host until it establishes a connection. |
```